### ISSUE TITLE
Error: undefined method `value' for "s(:splat,\n  s(:const, nil, :GROUPING_ATTRS))":RuboCop::AST::Node

### ISSUE THREAD

**Issue by alegrria** [rubocop-hq/rubocop#8269]
## Expected behavior
I expected rubocop to find either no offenses or some of them.

## Actual behavior
1 error occurred:
An error occurred while Style/BisectedAttrAccessor cop was inspecting <name_of_file.rb>

```
undefined method `value' for "s(:splat,\n  s(:const, nil, :GROUPING_ATTRS))":RuboCop::AST::Node
/Users/annakizilova/.rvm/gems/ruby-2.5.5/gems/rubocop-0.87.0/lib/rubocop/cop/style/bisected_attr_accessor.rb:50:in `map'
/Users/annakizilova/.rvm/gems/ruby-2.5.5/gems/rubocop-0.87.0/lib/rubocop/cop/style/bisected_attr_accessor.rb:50:in `block in accessor_names'
/Users/annakizilova/.rvm/gems/ruby-2.5.5/gems/rubocop-0.87.0/lib/rubocop/cop/style/bisected_attr_accessor.rb:49:in `each'
/Users/annakizilova/.rvm/gems/ruby-2.5.5/gems/rubocop-0.87.0/lib/rubocop/cop/style/bisected_attr_accessor.rb:49:in `accessor_names'
/Users/annakizilova/.rvm/gems/ruby-2.5.5/gems/rubocop-0.87.0/lib/rubocop/cop/style/bisected_attr_accessor.rb:27:in `on_class'
/Users/annakizilova/.rvm/gems/ruby-2.5.5/gems/rubocop-0.87.0/lib/rubocop/cop/commissioner.rb:91:in `block (2 levels) in trigger_responding_cops'
/Users/annakizilova/.rvm/gems/ruby-2.5.5/gems/rubocop-0.87.0/lib/rubocop/cop/commissioner.rb:113:in `with_cop_error_handling'
/Users/annakizilova/.rvm/gems/ruby-2.5.5/gems/rubocop-0.87.0/lib/rubocop/cop/commissioner.rb:90:in `block in trigger_responding_cops'
/Users/annakizilova/.rvm/gems/ruby-2.5.5/gems/rubocop-0.87.0/lib/rubocop/cop/commissioner.rb:89:in `each'
/Users/annakizilova/.rvm/gems/ruby-2.5.5/gems/rubocop-0.87.0/lib/rubocop/cop/commissioner.rb:89:in `trigger_responding_cops'
/Users/annakizilova/.rvm/gems/ruby-2.5.5/gems/rubocop-0.87.0/lib/rubocop/cop/commissioner.rb:61:in `block (2 levels) in <class:Commissioner>'
/Users/annakizilova/.rvm/gems/ruby-2.5.5/gems/rubocop-ast-0.1.0/lib/rubocop/ast/traversal.rb:59:in `block in on_begin'
/Users/annakizilova/.rvm/gems/ruby-2.5.5/gems/rubocop-ast-0.1.0/lib/rubocop/ast/traversal.rb:59:in `each'
/Users/annakizilova/.rvm/gems/ruby-2.5.5/gems/rubocop-ast-0.1.0/lib/rubocop/ast/traversal.rb:59:in `on_begin'
/Users/annakizilova/.rvm/gems/ruby-2.5.5/gems/rubocop-0.87.0/lib/rubocop/cop/commissioner.rb:62:in `block (2 levels) in <class:Commissioner>'
/Users/annakizilova/.rvm/gems/ruby-2.5.5/gems/rubocop-ast-0.1.0/lib/rubocop/ast/traversal.rb:101:in `on_class'
/Users/annakizilova/.rvm/gems/ruby-2.5.5/gems/rubocop-0.87.0/lib/rubocop/cop/commissioner.rb:62:in `block (2 levels) in <class:Commissioner>'
/Users/annakizilova/.rvm/gems/ruby-2.5.5/gems/rubocop-ast-0.1.0/lib/rubocop/ast/traversal.rb:59:in `block in on_begin'
/Users/annakizilova/.rvm/gems/ruby-2.5.5/gems/rubocop-ast-0.1.0/lib/rubocop/ast/traversal.rb:59:in `each'
/Users/annakizilova/.rvm/gems/ruby-2.5.5/gems/rubocop-ast-0.1.0/lib/rubocop/ast/traversal.rb:59:in `on_begin'
/Users/annakizilova/.rvm/gems/ruby-2.5.5/gems/rubocop-0.87.0/lib/rubocop/cop/commissioner.rb:62:in `block (2 levels) in <class:Commissioner>'
/Users/annakizilova/.rvm/gems/ruby-2.5.5/gems/rubocop-ast-0.1.0/lib/rubocop/ast/traversal.rb:14:in `walk'
/Users/annakizilova/.rvm/gems/ruby-2.5.5/gems/rubocop-0.87.0/lib/rubocop/cop/commissioner.rb:74:in `investigate'
/Users/annakizilova/.rvm/gems/ruby-2.5.5/gems/rubocop-0.87.0/lib/rubocop/cop/team.rb:151:in `investigate_partial'
/Users/annakizilova/.rvm/gems/ruby-2.5.5/gems/rubocop-0.87.0/lib/rubocop/cop/team.rb:83:in `investigate'
/Users/annakizilova/.rvm/gems/ruby-2.5.5/gems/rubocop-0.87.0/lib/rubocop/runner.rb:296:in `inspect_file'
/Users/annakizilova/.rvm/gems/ruby-2.5.5/gems/rubocop-0.87.0/lib/rubocop/runner.rb:246:in `block in do_inspection_loop'
/Users/annakizilova/.rvm/gems/ruby-2.5.5/gems/rubocop-0.87.0/lib/rubocop/runner.rb:278:in `block in iterate_until_no_changes'
/Users/annakizilova/.rvm/gems/ruby-2.5.5/gems/rubocop-0.87.0/lib/rubocop/runner.rb:271:in `loop'
/Users/annakizilova/.rvm/gems/ruby-2.5.5/gems/rubocop-0.87.0/lib/rubocop/runner.rb:271:in `iterate_until_no_changes'
/Users/annakizilova/.rvm/gems/ruby-2.5.5/gems/rubocop-0.87.0/lib/rubocop/runner.rb:242:in `do_inspection_loop'
/Users/annakizilova/.rvm/gems/ruby-2.5.5/gems/rubocop-0.87.0/lib/rubocop/runner.rb:122:in `block in file_offenses'
/Users/annakizilova/.rvm/gems/ruby-2.5.5/gems/rubocop-0.87.0/lib/rubocop/runner.rb:147:in `file_offense_cache'
/Users/annakizilova/.rvm/gems/ruby-2.5.5/gems/rubocop-0.87.0/lib/rubocop/runner.rb:120:in `file_offenses'
/Users/annakizilova/.rvm/gems/ruby-2.5.5/gems/rubocop-0.87.0/lib/rubocop/runner.rb:111:in `process_file'
/Users/annakizilova/.rvm/gems/ruby-2.5.5/gems/rubocop-0.87.0/lib/rubocop/runner.rb:90:in `block in each_inspected_file'
/Users/annakizilova/.rvm/gems/ruby-2.5.5/gems/rubocop-0.87.0/lib/rubocop/runner.rb:89:in `each'
/Users/annakizilova/.rvm/gems/ruby-2.5.5/gems/rubocop-0.87.0/lib/rubocop/runner.rb:89:in `reduce'
/Users/annakizilova/.rvm/gems/ruby-2.5.5/gems/rubocop-0.87.0/lib/rubocop/runner.rb:89:in `each_inspected_file'
/Users/annakizilova/.rvm/gems/ruby-2.5.5/gems/rubocop-0.87.0/lib/rubocop/runner.rb:78:in `inspect_files'
/Users/annakizilova/.rvm/gems/ruby-2.5.5/gems/rubocop-0.87.0/lib/rubocop/runner.rb:39:in `run'
/Users/annakizilova/.rvm/gems/ruby-2.5.5/gems/rubocop-0.87.0/lib/rubocop/cli/command/execute_runner.rb:21:in `execute_runner'
/Users/annakizilova/.rvm/gems/ruby-2.5.5/gems/rubocop-0.87.0/lib/rubocop/cli/command/execute_runner.rb:13:in `run'
/Users/annakizilova/.rvm/gems/ruby-2.5.5/gems/rubocop-0.87.0/lib/rubocop/cli/command.rb:10:in `run'
/Users/annakizilova/.rvm/gems/ruby-2.5.5/gems/rubocop-0.87.0/lib/rubocop/cli/environment.rb:17:in `run'
/Users/annakizilova/.rvm/gems/ruby-2.5.5/gems/rubocop-0.87.0/lib/rubocop/cli.rb:65:in `run_command'
/Users/annakizilova/.rvm/gems/ruby-2.5.5/gems/rubocop-0.87.0/lib/rubocop/cli.rb:72:in `execute_runners'
/Users/annakizilova/.rvm/gems/ruby-2.5.5/gems/rubocop-0.87.0/lib/rubocop/cli.rb:41:in `run'
/Users/annakizilova/.rvm/gems/ruby-2.5.5/gems/rubocop-0.87.0/exe/rubocop:13:in `block in <top (required)>'
/Users/annakizilova/.rvm/rubies/ruby-2.5.5/lib/ruby/2.5.0/benchmark.rb:308:in `realtime'
/Users/annakizilova/.rvm/gems/ruby-2.5.5/gems/rubocop-0.87.0/exe/rubocop:12:in `<top (required)>'
/Users/annakizilova/.rvm/gems/ruby-2.5.5/bin/rubocop:23:in `load'
/Users/annakizilova/.rvm/gems/ruby-2.5.5/bin/rubocop:23:in `<main>'
/Users/annakizilova/.rvm/gems/ruby-2.5.5/bin/ruby_executable_hooks:24:in `eval'
/Users/annakizilova/.rvm/gems/ruby-2.5.5/bin/ruby_executable_hooks:24:in `<main>'
```

## Steps to reproduce the problem
We have following for the GROUPING_ATTRS in our code
`GROUPING_ATTRS = %i[first second third]`

## RuboCop version
0.87.0 (using Parser 2.7.1.4, rubocop-ast 0.1.0, running on ruby 2.5.5 x86_64-darwin18)

**Comment by bbatsov** [comment]
Please, try RuboCop 0.87.1. I think we fixed this bug there.

**Comment by alegrria** [comment]
Yes, updating to 0.87.1 removed the error.
Thank you!
